### ISSUE TITLE
Beaming in Tempo Indication

### ISSUE THREAD

**Issue by arshiacont** [grame-cncm/guidolib#152]
The [Guido tempo example](https://guidodoc.grame.fr/examples/tempo/) clearly show the possibility of indicating `[1/8]=[1/8]` to show tempo indication using notation in `tempo` tag.

Is there currently a way to achieve a tempo notation such as this one?
<img width="229" alt="image" src="https://user-images.githubusercontent.com/3298204/165744877-8d8eada2-a011-48f8-b7ce-528b8b4ef4d4.png">

**Comment by dfober** [comment]
No, you can use as many symbols as you want e.g. \tempo<"Swing [1/8] [1/8] = [1/4] [1/8]"> but there is no way do indicate beaming or irregular division. Currently, only single note indications are decoded.

**Comment by zhwei8090** [comment]
We also have such a need, but it was rejected by me
